docs/user/installation_and_configuration.rst

```diff
@@ -459,3 +475,12 @@ is printed out when you start GeoServer::
     ----------------------------------
     - GEOSERVER_DATA_DIR: /opt/devel/install/geoserver-data-dir
     ----------------------------------
+
+Note about Apache Commons Collections
```

Comment:
@mdzimmerman, I tried to help;)  Feel free to fix the formatting if it is messed up or can be improved.